helm/values.yaml

```diff
@@ -148,6 +148,7 @@ automaticChildApproval:
   enabled: true
 
 # debug: true
+defaultLogLevel: INFO
```

Comment:
```suggestion
# The "logLevel" configuration option adds a partial redundancy to the `debug` setting.
# Removing the `debug` setting is a breaking change though - we are going to remove the `debug` setting in the context of a larger refactoring to avoid multiple breaking releases.
# logLevel: INFO
```

Reply:
Indeed, `debug` and `logLevel` are confusing and redundant. Removing it would be a breaking change.
That's the reason I suggested `defaultLogLevel` which implicitly means the logLevel if not override by `debug` mode.
But, it depends on the breaking change management strategy.

Reply:
Yeah, we discussed that internally and want to remove the `debug` option with the next breaking change - and decided to add an explanatory comment to reflect that :)